src/api/endpoint/api/stars.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @prettier
```

Comment:
Should this pragma be removed?

Reply:
Actually it is one of my comments above in the What's next section.